entities/OrderDetails.java

```diff
@@ -0,0 +1,72 @@
+package entities;
+
+import constants.Status;
+
+import java.time.Instant;
+
+public class OrderDetails extends Base {
+
+    //Attributes
+    private Integer orderID;
+    private Integer productID;
```

Comment:
You can refer to the whole `Product` entity

Reply:
donde